packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -482,6 +503,40 @@ export default defineComponent({
             width: 'expand',
             wrap: 'truncate'
           },
+
+          {
+            name: 'manager',
+            prop: 'spaceRoles',
```

Comment:
I believe you don't need the property `prop` when the `type` is set to `slot`. This goes for the for the following fields as well.

Reply:
yeah we added the the prop attribute because there is an filter that removes every field by name if its not in the firstResource object, and things like quotaUsed are accessed via quota.used